### ISSUE TITLE
Cannot install iqsharp

### ISSUE THREAD

**Issue by harshitgarg22** [microsoft/iqsharp#147]
**Describe the bug**
While installing dotnet iqsharp through `dotnet iqsharp install`, I get an error saying 
> The required library libhostfxr.so could not be found. 
If this is a self-contained application, that library should exist in [/home/$USERNAME/.dotnet/tools/.store/microsoft.quantum.iqsharp/0.11.2004.2825/microsoft.quantum.iqsharp/0.11.2004.2825/tools/netcoreapp3.1/any/].

I made a search on / for the file, found it in the snap installation of dotnet-sdk at `/snap/dotnet-sdk/86/host/fxr/3.1.4`, pasted it in the above mentioned folder. Now I get another error when I run `dotnet iqsharp install` again.

> It was not possible to find any compatible framework version
The framework 'Microsoft.AspNetCore.App', version '3.1.0' was not found.
  No frameworks were found.
You can resolve the problem by installing the specified framework and/or SDK.
The specified framework can be found at:
  https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=3.1.0&arch=x64&rid=solus.4.1-x64

Since there is no direct package, I have to use linux binaries. Snap only has 3.1.4. Binaries only has minimum 3.1.0. I think this shouldn't be a problem since the docs mention .NET Core SDK 3.1?

**To Reproduce**
1. Run `dotnet tool install -g Microsoft.Quantum.IQSharp`
2. Run `dotnet iqsharp install`
3. See error

**Expected behavior**
> Installed successfully!

**System information**

dotnet --info returns

> .NET Core SDK (reflecting any global.json):
 Version:   3.1.300
 Commit:    b2475c1295
Runtime Environment:
 OS Name:     solus
 OS Version:  4.1
 OS Platform: Linux
 RID:         linux-x64
 Base Path:   /snap/dotnet-sdk/86/sdk/3.1.300/
Host (useful for support):
  Version: 3.1.4
  Commit:  0c2e69caa6
.NET Core SDKs installed:
  3.1.300 [/snap/dotnet-sdk/86/sdk]
.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.1.4 [/snap/dotnet-sdk/86/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.4 [/snap/dotnet-sdk/86/shared/Microsoft.NETCore.App]

**Additional context**
I'm trying to install iqsharp for working with jupyter notebooks (Quantum Katas to be specific), for the codeforces challenge.

**Comment by harshitgarg22** [comment]
Alright so I went ahead searched the entire internet for a solution, and dotnet-iqsharp at least installs now.

The problem was in my installation of iqsharp (which is apparent when even `dotnet-cowsay` can't find the framework). The problem was, the `dotnet-install.sh` script saves the dotnet installation in ~/.dotnet, and for some obscure reason the .NET applications seem to be referring to /usr/share to find the dotnet installation, even though `$DOTNET_ROOT` is set to `~/.dotnet`. So I copied the dotnet folder `cp -r ~/.dotnet /usr/share/`. And after uninstalling and installing iqsharp as a dotnet tool, voila! `dotnet iqsharp install` works! The Q# notebooks for `index.ipynb` from Quantum Katas work now!
**NB** Instead of copying the dotnet folder, it'd be wiser to create a symlink to your snap installation or whatever, to handle updates and keep everything together.

Just putting it up here in case someone else is going through tough times.
